phpBB/phpbb/cache/driver/file.php

```diff
@@ -584,4 +587,19 @@ function _write($filename, $data = null, $expires = 0, $query = '')
 
 		return $return_value;
 	}
+
+	/**
+	* Change the cache path name
+	*
+	* @param string $name filename
+	* @return string the corrected path
+	*/
+	function get_file_name($name)
```

Comment:
Well it's not for retrieving the filename but rather for replacing the slashes in the file name?